ha/setup/ha_setup.py

```diff
@@ -323,6 +324,20 @@ def _update_env(self, node_name: str, node_type: str, cluster_type: str) -> None
         Log.info("CONFIG: Update ha configuration files")
         Conf.save(const.HA_GLOBAL_INDEX)
 
+    def _fetch_fids(self) -> None:
+        """
+        Fetch fids from hare and store in a config file
+        """
+        try:
+            Log.info("Fetch fids from hare and store in a conf file")
+            fids_output, err, rc = self._execute.run_cmd(const.HCTL_FETCH_FIDS, check_error=True)
```

Comment:
we are not planning to store fids in Consul.

Reply:
Not now, but later yes.

Reply:
@ajaysrivas I will merge this PR, please let me know if there are further comments, we can handle those separately.

Reply:
Created https://jts.seagate.com/browse/EOS-19248 for storing the FIDs in consul.